src/services/fileStorage/proxy-service.js

```diff
@@ -430,7 +432,8 @@ const signedUrlService = {
 			throw new NotFound('File seems not to be there.');
 		}
 
-		const creatorId = fileObject.permissions[0].refPermModel !== 'user' ? userId : fileObject.permissions[0].refId;
+		const creatorId = fileObject.creator
+			|| fileObject.permissions[0].refPermModel !== 'user' ? userId : fileObject.permissions[0].refId;
```

Comment:
please add a comment that this is backwards compability and may be removed in a future update